packages/datagrid/src/components/DefaultCellRenderer/DefaultRenderer.component.js

```diff
@@ -1,8 +1,18 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+import DefaultValueRenderer from './DefaultValueRenderer.component';
+import FormatValue, { REG_EXP_HAS_WHITE_SPACE_CHARACTERS } from './FormatValue.component';
+
 export default function DefaultRenderer({ data }) {
-	return <span>{data.value}</span>;
+	const HIDDEN_CHARACTERS_REG_EXP = new RegExp(REG_EXP_HAS_WHITE_SPACE_CHARACTERS);
+	const hiddenCharsRegExpMatch = data.value.match(HIDDEN_CHARACTERS_REG_EXP);
```

Comment:
Here you re-implement part of the logic of FormatValue.
IMO you should just use FormatValue, that should manage the case where there is no special char it want to replace

Reply:
It is more complicated. This logic is used to know if I show a DefaultValueRenderer with a complex renderer or just a label for the DefaultValueRenderer. It is for this reason i have set this here. So, with your remarks i change that to use a function from FormatValue to detect white characters